frame/assets/src/impl_fungibles.rs

```diff
@@ -65,69 +77,24 @@ impl<T: Config<I>, I: 'static> fungibles::Inspect<<T as SystemConfig>::AccountId
 	}
 }
 
-impl<T: Config<I>, I: 'static> fungibles::InspectMetadata<<T as SystemConfig>::AccountId>
+impl<T: Config<I>, I: 'static> fungibles::Mutate<<T as SystemConfig>::AccountId> for Pallet<T, I> {}
+impl<T: Config<I>, I: 'static> fungibles::Balanced<<T as SystemConfig>::AccountId>
 	for Pallet<T, I>
 {
-	/// Return the name of an asset.
-	fn name(asset: &Self::AssetId) -> Vec<u8> {
```

Comment:
What's the reason for removing those methods? This could easily break existing integrations, not to say now there are no public methods to get that info on the pallet level (e.g. another pallet would need to create an asset and then read/validate the name/symbol/decimals)

Reply:
Metadata can be inspected and mutated through the traits in the metadata module. I apparently never added the file - will do so now.

Reply:
Sorry @gavofyork , what `metadata` module you're referencing to? If it's the Metadata storage item, then it's not public and thus can't be accessed from another pallet.